source/class/tokenfield/Token.js

```diff
@@ -712,10 +712,10 @@ qx.Class.define('tokenfield.Token',
     },
 
     /**
-     * Removes a token from the selection id it matches the data label.
+     * Removes a token from the selection if it matches the data label.
      * @param data {Map} model
      */
-    removeItem: function (data) {
+    deselectItem: function (data) {
```

Comment:
I have not probem with the method name change, but that might break things for existing users of the contrib. Can you add a BC method stub `removeItem: this.deselectItem`, please?

Reply:
As this PR added the method, I am the only user of this feature ;-)

Reply:
Ok ;-) - It's been a while since I worked on this code (I am not even the original author).